doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,478 @@
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.0 requires Clear Linux OS version 29070 or newer. Please follow the 
+instructions in the Getting started guide for Intel NUC.
+
+Version 1.0 new features
+************************
+
+VirtHardware Support
+====================
+Apollo Lake NUC, Apollo Lake UP Squared (UP2) board with serial port and 
+Kaby Lake NUC are supported. (See Supported Hardware for supported platform 
+details.)
+
+APL UP2 board with SBL firmware: ACRN supports APL UP2 board with Slim 
+Bootloader (SBL) firmware. Slim Bootloader is a modern, flexible, 
+light-weight, open source reference bootloader that is also  fast, small, 
+customizable, and secure. An end-to-end reference build has been verified 
+on UP2/SBL board using  ACRN hypervisor, Clear Linux OS as SOS, and Clear 
+Linux OS as UOS. See the Using SBL on UP2 Board documentation for 
+step-by-step instructions.
+
+VT-x, VT-d
+==========
+Based on Intel VT-x virtualization technology, ACRN emulates a virtual CPU 
+with core partition and simple scheduling. VT-d provides hardware support 
+for isolating and restricting device accesses to only the owner of the 
+partition managing the device. It allows assigning I/O devices to a VM, 
+and extending the protection and isolation properties of VMs for 
+I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+==============================
+ACRN hypervisor supports virtualized APIC-V, EPT, IOAPIC, and LAPIC functionality.
+
+One E2E binary to support all UEFI platform: ACRN can support both Apollo Lake 
+(APL) and Kaby Lake (KBL) NUCs. Instead of having separate builds, this release 
+offers community developers a single end-to-end reference build that supports 
+both UEFI hardware platforms, configured with a new boot parameter. See Getting 
+Started Guides for more information.
+
+Enable Privileged VM support for real-time UOS in ACRN: Initial patches to enable 
+a User OS (UOS) running as a virtual machine (VM) with real-time characteristics, 
+also called a “Privileged VM”. We’ve published a tutorial Using PREEMPT_RT-Linux 
+for real-time UOS. More patches for ACRN real time support will continue.
+
+Enable cache QOS with CAT
+=========================
+Cache Allocation Technology (CAT) is avaible on Apollo Lake (APL) platforms, 
+providing cache isolation between VMs mainly for real-time performance quality 
+of service (QoS). The CAT for a specific VM is normally set up at boot time per 
+the VM configuration determined at build time. For debugging and performance 
+tuning, the CAT can also be enabled and configured at runtime by writing proper 
+values to certain MSRs using the wrmsr command on ACRN shell.
+
+Enable QoS based on runC container
+==================================
+ACRN supports Device-Model QoS based on runC container to control the SOS 
+resources (CPU, Storage, MEM, NET) by modifying the runC configuration file.
```

Comment:
We don't seem to have any documentation about this feature, can we add some for the 1.0 release. It would be important in my opinion since it's called out here.